README.md

```diff
@@ -25,8 +25,13 @@ var inlineBase64 = require('nodemailer-plugin-inline-base64');
 #### 2. Attach it as a 'compile' handler for a nodemailer transport object
 
 ```javascript
-nodemailerTransport.use('compile', inlineBase64)
+nodemailerTransport.use('compile', inlineBase64(options))
 ```
+Options allow to set CID prefix<sup><a href="#1">1</a></sup> ```{cidPrefix: 'somePrefix_'}```,
+then all inline images will have prefix in cid, i.e.: `cid:somePrefix_5fe3b631c651bdb1`. If you don't need this,
+you can use inlineBase64 plugin without options.
```

Comment:
Please fix the code sample in https://github.com/vologab/nodemailer-plugin-inline-base64/blob/7e1372011293fc4ddebf859558c51110b8cf3811/README.md#example also